packages/taquito-michelson-encoder/test/tokens/timestamp.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TimestampToken } from '../../src/tokens/comparable/timestamp';
+
+describe('Timestamp token', () => {
+  let token: TimestampToken;
+
+  beforeEach(() => {
+    token = new TimestampToken({ prim: 'timestamp', args: [{ string: '1638554817'}], annots: [] }, 0, null as any);
+  });
+
+  describe('Execute', () => {
+    it('Should return the RFC3339 time format when type is int and value is UNIX timestamp', () => {
+      const result = token.Execute({ int: 1638555790 } as any);
+      expect(result).toEqual('2021-12-03T18:23:10.000Z');
+    });
+
+    it('Should return the RFC3339 time format when type is string and value is UNIX timestamp', () => {
+      const result = token.Execute({ string: '1638555790' } as any);
```

Comment:
I don't think there is a need to cast to any here as the expected type of the parameter is `{ string?: string; int?: string }`.